app/models/container_project.rb

```diff
@@ -69,6 +71,14 @@ def perf_rollup_parents(interval_name = nil)
     []
   end
 
+  def all_computer_system_ids
+    return computer_systems.ids
```

Comment:
no need for the return

Reply:
Thanks, just found how it's done in ems :-) fixed